src/components/TouristInfoCards.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+var TouristCardsLinksAndImages = [
+  {
+    CityName: "Glasgow",
+    Link: "https://peoplemakeglasgow.com/",
+    Photo: "https://peoplemakeglasgow.com/templates/people/images/logo.png"
+  },
+  {
+    CityName: "Manchester",
+    Link: "https://www.visitmanchester.com/",
+    Photo: "https://www.visitmanchester.com/dbimgs/logo(4).png"
+  },
+  {
+    CityName: "London",
+    Link: "https://www.visitlondon.com/",
+    Photo: "https://cdn.londonandpartners.com/fl/assets/33374-310x174.jpg"
+  }
+];
+const TouristInfoCards = props => {
+  return (
+    <div className="Card-Container">
```

Comment:
Try to follow one unique naming convention in your code. So here for the className value, as you use snakecase format in the rest of the file, you could rename it `card-container` instead of `Card-Container` for consistency.

Reply:
sometimes the uppercase letters is added unconsciously :D